service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapper.java

```diff
@@ -114,13 +116,28 @@ private String buildAvailabilityTime(ProcedureRequest procedureRequest) {
             getEarliestRecallDate(procedureRequest),
             getRequester(procedureRequest),
             getReasonCode(procedureRequest),
-            getNotes(procedureRequest)
+            getNotes(procedureRequest),
+            getSupportingInformation(procedureRequest)
         )
             .filter(Optional::isPresent)
             .map(Optional::get)
             .collect(Collectors.joining(StringUtils.SPACE)));
     }
 
+    private Optional<String> getSupportingInformation(ProcedureRequest procedureRequest) {
+        if (procedureRequest.hasSupportingInfo()) {
+            return Optional.of(procedureRequest.getSupportingInfo().stream()
+                .filter(this::checkIfReferenceIsObservation)
+                .map((observationReference) -> extractObservation(messageContext, observationReference))
+                .collect(Collectors.joining(COMMA)));
+        }
+        return Optional.empty();
+    }
+
+    private boolean checkIfReferenceIsObservation(Reference reference) {
+        return reference.getReferenceElement().getResourceType().equals(ResourceType.Observation.name());
```

Comment:
It's usually better to compare a const with a variable rather than the opposite.